bibliopixel/remote/control.py

```diff
@@ -104,13 +110,32 @@ def __init__(self, layout, animations,
         self.handlers = {
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
-            'get_config': self.get_config
+            'get_config': self.get_config,
+            'trigger_animation': self.run_animation
         }
 
+        self.trigger_procs = {}
+        self.triggers = {}
+        for t in triggers:
+            typename = t['typename']
+            importer.import_symbol(typename)  # attempt early to fail early
+            if typename not in self.triggers:
+                self.triggers[typename] = []
+            t.pop('typename')
+            self.triggers[typename].append(t)
+
+        for typename, configs in self.triggers.items():
+            trigger_args = (typename, self.q_recv, configs)
+            self.trigger_procs[typename] = multiprocessing.Process(target=trigger_process.run_trigger,
```

Comment:
Does this have to be a process?

Since you're using your own queue, I could simplify `bibliopixel.threads.sub.run` to your needs here, perhaps in a later pull request - then you could switch between threading and mp.